components/badge.css

```diff
@@ -0,0 +1,52 @@
+.display-badge{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
```

Comment:
You use the same property in the avatar, you could make this utility class and you wherever required

Reply:
Currently, I am not following the utility first approach but I will use it once I start writing grid components. Thanks for the suggestion!